research/object_detection/core/region_similarity_calculator.py

```diff
@@ -35,7 +35,8 @@
 class RegionSimilarityCalculator(six.with_metaclass(ABCMeta, object)):
   """Abstract base class for region similarity calculator."""
 
-  def compare(self, boxlist1, boxlist2, scope=None):
+  def compare(self, boxlist1, boxlist2, scope=None,
+              groundtruth_labels=None, predicted_labels=None):
```

Comment:
groundtruth_labels and predicted_labels can be fields inside boxlist. We don't need to add additional arguments to the interface.